--- xen-orchestra/packages/xo-web/src/xo-app/index.js.orig	2020-05-27 07:32:11.288629863 +0200
+++ xen-orchestra/packages/xo-web/src/xo-app/index.js	2020-05-27 07:54:50.854676288 +0200
@@ -25,7 +25,6 @@
 import Dashboard from './dashboard'
 import Home from './home'
 import Host from './host'
-import Hub from './hub'
 import Jobs from './jobs'
 import Menu from './menu'
 import Modal, { alert, FormModal } from 'modal'
@@ -104,7 +103,6 @@
   xoa: Xoa,
   xosan: Xosan,
   import: Import,
-  hub: Hub,
   proxies: Proxies,
 })
 @connectStore(state => {
@@ -132,19 +130,6 @@
     const now = Math.floor(Date.now() / 1e3)
     const oneWeekAgo = now - 7 * 24 * 3600
     if (!previousDisclaimer || previousDisclaimer < oneWeekAgo) {
-      alert(
-        _('disclaimerTitle'),
-        <div>
-          <p>{_('disclaimerText1')}</p>
-          <p>
-            {_('disclaimerText2')}{' '}
-            <a href='https://xen-orchestra.com/#!/xoa?pk_campaign=xoa_source_upgrade&pk_kwd=ossmodal'>
-              XOA (turnkey appliance)
-            </a>
-          </p>
-          <p>{_('disclaimerText3')}</p>
-        </div>
-      )
       cookies.set('previousDisclaimer', now)
     }
   }
@@ -251,27 +236,6 @@
                   </Link>
                 </div>
               )}
-              {plan === 'Community' && !this.state.dismissedSourceBanner && (
-                <div className='alert alert-danger mb-0'>
-                  <a
-                    href='https://xen-orchestra.com/#!/xoa?pk_campaign=xo_source_banner'
-                    rel='noopener noreferrer'
-                    target='_blank'
-                  >
-                    {_('disclaimerText3')}
-                  </a>{' '}
-                  <a
-                    href='https://xen-orchestra.com/docs/installation.html#banner-and-warnings'
-                    rel='noopener noreferrer'
-                    target='_blank'
-                  >
-                    {_('disclaimerText4')}
-                  </a>
-                  <button className='close' onClick={this.dismissSourceBanner}>
-                    &times;
-                  </button>
-                </div>
-              )}
               <div style={CONTAINER_STYLE}>
                 <Shortcuts
                   name='XoApp'

