--- xen-orchestra/packages/xo-web/src/xo-app/home/host-item.js.orig	2023-11-23 15:10:46.973632453 +0100
+++ xen-orchestra/packages/xo-web/src/xo-app/home/host-item.js	2023-11-23 16:35:55.172162321 +0100
@@ -107,7 +107,7 @@
 
     if (getXoaPlan() === SOURCES) {
       message = 'poolSupportSourceUsers'
-      level = 'warning'
+      level = 'success'
     }
 
     if (supportLevel === 'total') {
@@ -168,29 +168,6 @@
         })
       }
 
-      if (hasLicenseRestrictions(host)) {
-        alerts.push({
-          level: 'danger',
-          render: (
-            <span>
-              <Icon icon='alarm' /> {_('licenseRestrictionsModalTitle')} {LICENSE_WARNING_BODY}
-            </span>
-          ),
-        })
-      }
-
-      const proSupportStatus = this._getProSupportStatus()
-      if (proSupportStatus !== undefined && proSupportStatus.level !== 'success') {
-        alerts.push({
-          level: proSupportStatus.level,
-          render: (
-            <span>
-              {proSupportStatus.icon} {_(proSupportStatus.message)}
-            </span>
-          ),
-        })
-      }
-
       if (!host.hvmCapable) {
         alerts.push({
           level: 'warning',
@@ -243,9 +220,6 @@
                 &nbsp;
                 <BulkIcons alerts={this._getAlerts()} />
                 &nbsp;
-                {proSupportStatus?.level === 'success' && (
-                  <Tooltip content={_(proSupportStatus.message)}>{proSupportStatus.icon}</Tooltip>
-                )}
               </EllipsisContainer>
             </Col>
             <Col mediumSize={3} className='hidden-lg-down'>
