--- xen-orchestra/packages/xo-web/src/xo-app/home/host-item.js.orig	2023-11-23 15:10:46.973632453 +0100
+++ xen-orchestra/packages/xo-web/src/xo-app/home/host-item.js	2023-11-23 16:35:55.172162321 +0100
@@ -107,7 +107,7 @@
 
     if (getXoaPlan() === SOURCES) {
       message = 'poolSupportSourceUsers'
-      level = 'warning'
+      level = 'success'
     }
 
     if (supportLevel === 'total') {
@@ -243,9 +220,6 @@
                 &nbsp;
                 <BulkIcons alerts={this._getAlerts()} />
                 &nbsp;
-                {proSupportStatus?.level === 'success' && (
-                  <Tooltip content={_(proSupportStatus.message)}>{proSupportStatus.icon}</Tooltip>
-                )}
               </EllipsisContainer>
             </Col>
             <Col mediumSize={3} className='hidden-lg-down'>
