--- xen-orchestra/packages/xo-web/src/xo-app/home/pool-item.js.orig	2025-01-24 22:16:48.704472898 +0100
+++ xen-orchestra/packages/xo-web/src/xo-app/home/pool-item.js	2025-01-25 20:18:45.882123118 +0100
@@ -81,9 +81,6 @@
       reacletteState.poolLicenseInfoByPoolId[pool.id]
     let tooltip = _('poolNoSupport')
 
-    if (getXoaPlan() === SOURCES) {
-      tooltip = _('poolSupportSourceUsers')
-    }
     if (supportLevel === 'total') {
       tooltip = _('earliestExpirationDate', {
         dateString:
@@ -177,8 +174,6 @@
                   </span>
                 )}
                 &nbsp;
-                {isAdmin && this._isXcpngPool() && supportLevel === 'total' && icon(this._getPoolLicenseIconTooltip())}
-                &nbsp;
                 {pool.HA_enabled && (
                   <span>
                     &nbsp;&nbsp;
