--- xen-orchestra/packages/xo-web/src/xo-app/menu/index.js.orig	2023-12-29 12:54:53.405589046 +0100
+++ xen-orchestra/packages/xo-web/src/xo-app/menu/index.js	2023-12-29 12:56:58.018920891 +0100
@@ -332,39 +332,6 @@
           },
         ],
       },
-      {
-        to: isAdmin ? 'xoa/update' : 'xoa/notifications',
-        icon: 'menu-xoa',
-        label: 'xoa',
-        extra: [
-          !isAdmin || xoaState === 'upToDate' ? null : <UpdateTag key='update' />,
-          noNotifications ? null : <NotificationTag key='notification' />,
-        ],
-        subMenu: [
-          isAdmin && {
-            to: 'xoa/update',
-            icon: 'menu-update',
-            label: 'updatePage',
-            extra: <UpdateTag />,
-          },
-          isAdmin && {
-            to: 'xoa/licenses',
-            icon: 'menu-license',
-            label: 'licensesPage',
-          },
-          {
-            to: 'xoa/notifications',
-            icon: 'menu-notification',
-            label: 'notificationsPage',
-            extra: <NotificationTag />,
-          },
-          isAdmin && {
-            to: 'xoa/support',
-            icon: 'menu-support',
-            label: 'supportPage',
-          },
-        ],
-      },
       isAdmin && {
         to: '/settings/servers',
         icon: 'menu-settings',
@@ -441,44 +408,6 @@
           },
         ],
       },
-      isAdmin && {
-        to: '/hub/templates',
-        icon: 'menu-hub',
-        label: 'hubPage',
-        subMenu: [
-          {
-            to: '/hub/templates',
-            icon: 'hub-template',
-            label: 'templatesLabel',
-          },
-          {
-            to: '/hub/recipes',
-            icon: 'hub-recipe',
-            label: 'recipesLabel',
-          },
-        ],
-      },
-      isAdmin && {
-        to: '/proxies',
-        icon: 'proxy',
-        label: 'proxies',
-        extra: [
-          this._areProxiesOutOfDate() ? (
-            <Tooltip content={_('proxiesNeedUpgrade')}>
-              <StackedIcons
-                icons={[
-                  { color: 'text-success', icon: 'circle', size: 2 },
-                  { icon: 'menu-update', size: 1 },
-                ]}
-              />
-            </Tooltip>
-          ) : nProxiesErrors > 0 ? (
-            <Tooltip content={_('someProxiesHaveErrors', { n: nProxiesErrors })}>
-              <span className='tag tag-pill tag-danger'>{nProxiesErrors}</span>
-            </Tooltip>
-          ) : null,
-        ],
-      },
       isAdmin && { to: '/about', icon: 'menu-about', label: 'aboutPage' },
       {
         to: '/tasks',
@@ -486,11 +415,6 @@
         label: 'taskMenu',
         pill: nResolvedTasks,
       },
-      isAdmin && {
-        to: '/xostor',
-        label: 'xostor',
-        icon: 'menu-xostor',
-      },
       !noOperatablePools && {
         to: '/import/vm',
         icon: 'menu-new-import',
