--- xen-orchestra/packages/xo-web/src/xo-app/menu/index.js.orig	2021-03-31 17:05:53.733482570 +0200
+++ xen-orchestra/packages/xo-web/src/xo-app/menu/index.js	2021-03-31 17:09:38.342579089 +0200
@@ -267,39 +267,6 @@
           },
         ],
       },
-      {
-        to: isAdmin ? 'xoa/update' : 'xoa/notifications',
-        icon: 'menu-xoa',
-        label: 'xoa',
-        extra: [
-          !isAdmin || xoaState === 'upToDate' ? null : <UpdateTag key='update' />,
-          noNotifications ? null : <NotificationTag key='notification' />,
-        ],
-        subMenu: [
-          isAdmin && {
-            to: 'xoa/update',
-            icon: 'menu-update',
-            label: 'updatePage',
-            extra: <UpdateTag />,
-          },
-          isAdmin && {
-            to: 'xoa/licenses',
-            icon: 'menu-license',
-            label: 'licensesPage',
-          },
-          {
-            to: 'xoa/notifications',
-            icon: 'menu-notification',
-            label: 'notificationsPage',
-            extra: <NotificationTag />,
-          },
-          isAdmin && {
-            to: 'xoa/support',
-            icon: 'menu-support',
-            label: 'supportPage',
-          },
-        ],
-      },
       isAdmin && {
         to: '/settings/servers',
         icon: 'menu-settings',
@@ -376,28 +343,6 @@
           },
         ],
       },
-      isAdmin && {
-        to: '/hub/templates',
-        icon: 'menu-hub',
-        label: 'hubPage',
-        subMenu: [
-          {
-            to: '/hub/templates',
-            icon: 'hub-template',
-            label: 'templatesLabel',
-          },
-          {
-            to: '/hub/recipes',
-            icon: 'hub-recipe',
-            label: 'recipesLabel',
-          },
-        ],
-      },
-      isAdmin && {
-        to: '/proxies',
-        icon: 'proxy',
-        label: 'proxies',
-      },
       isAdmin && { to: '/about', icon: 'menu-about', label: 'aboutPage' },
       !noOperatablePools && {
         to: '/tasks',
@@ -405,7 +350,6 @@
         label: 'taskMenu',
         pill: nTasks,
       },
-      isAdmin && { to: '/xosan', icon: 'menu-xosan', label: 'xosan' },
       !noOperatablePools && {
         to: '/import/vm',
         icon: 'menu-new-import',
